deformable_manipulation_experiment_params/include/deformable_manipulation_experiment_params/ros_params.hpp

```diff
@@ -1169,6 +1226,11 @@ namespace smmap
         return ROSHelpers::GetParam(nh, "sampling_controller/max_sampling_counts", 1000);
     }
 
+    inline bool GetFixStepSize(ros::NodeHandle& nh)
```

Comment:
We should change this function name to be more meaningful, I have no idea what "fix step size" means. One solution might be to add comments rather than change the function name.

Reply:
It is the norm of each sampled grippers motion

Reply:
Lets make it more clear in the code what is going on. A bool is not a norm.